products/microos/main.pm

```diff
@@ -91,6 +91,15 @@ sub load_installation_tests {
     }
 }
 
+sub load_security_tests_selinux {
+    # SELinux tests for Micro OS operating system
+    # ALWAYS run following tests in sequence because of the dependencies
+    loadtest 'security/selinux/sestatus';
+    loadtest 'security/selinux/selinux_smoke';
+    # Change SELinux from "permissive" mode to "enforcing" mode for testing
+    loadtest 'security/selinux/enforcing_mode_setup';
```

Comment:
The problem with this is that it will test enforcing mode, when the whole exercise on MicroOS is intended to use 'targetted'

after all, we're going to have selinux-targeted-setup installed by default
https://build.opensuse.org/package/view_file/openSUSE:Factory/selinux-targeted-setup/selinux-targeted-setup.spec?expand=1

So I think this line is not what we want :)

Reply:
Thank you. Just to clarify - "targeted" is a policy type, not selinux mode. In other words we need to test targeted policy with either enforcing or permissive mode. I added it 'cos it tests booting with enforcing mode. But you are right, this is not what we want right now as the current tests works only with "minimum" policy type not "targeted", so we need to modify them the ```if is_microos``` way you suggested below.